central/plugins/org.jboss.tools.central/src/org/jboss/tools/central/internal/browser/CentralBrowserErrorWrapper.java

```diff
@@ -78,10 +72,14 @@ public void run() {
 			}
 		});
 		
+		//report that browser is not available
+		UsageEventType eventType = JBossCentralActivator.getDefault().getUsedBrowserEventType();
+		UsageReporter.getInstance().trackEvent(eventType.event("N/A")); //$NON-NLS-1$
+		
 		parent.getParent().layout(true, true);
```

Comment:
shouldn't we ensure the layout stuff is done before calling the event in case something bad happens ?